src/ui/views/chrome.jade

```diff
@@ -12,5 +12,5 @@ html(lang='en')
     title Kibana
     block head
   body(kbn-chrome, id='#{appName}-body')
-    script window.__KBN__ = !{j(kibanaPayload)};
+    meta(kbn-initial-state=JSON.stringify(kibanaPayload))
```

Comment:
I like it, but I think we should use the meta tag how it's supposed to be use semantically:

 - in the `<head>`
 - with `name="kbn-initial-state"`
 - with `content="..."`

http://www.w3schools.com/tags/tag_meta.asp